exercises/webapp/src/components/NewsForm.stories.js

```diff
@@ -0,0 +1,16 @@
+import NewsForm from './NewsForm';
+
+export default {
+  title: 'NewsForm',
+  component: NewsForm,
+  argTypes: {},
+};
+
+const Template = (args, {argTypes}) => ({
+  props: Object.keys(argTypes),
+  components: {NewsForm},
+  template: '<NewsForm @onClick="onClick" v-bind="$props" />',
+});
```

Comment:
You can also pack your methods here, for example create:  methods: { create: action("create") },